_data/banking.yml

```
@@ -222,6 +222,15 @@ websites:
     facebook: bbvacompass
     twitter: BBVACompass
 
+  - name: Bendigo Bank
+    url: https://www.bendigobank.com.au
+    img: bendigobank.png
+    tfa:
+      - proprietary
+    doc: https://www.bendigobank.com.au/help/security/online-security/e-tokens/
```

Let's list it as non-supporting then.
```suggestion
    facebook: BendigoBank 
    twitter: bendigobank
```